share/etc/oned.conf

```diff
@@ -1388,3 +1388,47 @@ AUTH_MAD_CONF = [
     MAX_TOKEN_TIME = "-1"
 ]
 
+VN_MAD_CONF = [
```

Comment:
Add comments to oned.conf explaining what is VN_MAD_CONF and valid attributes to be defined, right now just NAME and BRIDGE_TYPE